### DISCUSSION TITLE
Improve datepicker to make selecting date ranges more intuitive

### DISCUSSION THREAD

**Discussion by Puma-HC-Stirn** [davish/obsidian-full-calendar#385] [Feature Requests]
If an "all-day-event" has a duration of several days, the time block in the calendar does not cover the last day.

Example:

title: Snow and Fun
allDay: true
startTime: 
endTime: 
date: 2022-12-26
endDate: 2023-01-02

In the calendar, the event ist shown on 2023-01-01, but not on 2023-01-02. This is true in every view (monthly, weekly or daily view), and even in the list view.

I expected the event to be shown even on its last day of duration.

I have the newest update of Obsidian (v0.10.6) and Full calendar (0.6.1) at this time.

**Comment by davish** [comment]
This is not a bug per se, but just a consequence of the date range being exclusive in the underlying [FullCalendar plugin](https://fullcalendar.io/docs/event-parsing). This would be fixed with a better datepicker that had support for date ranges. I'll edit the title and change to a UI enhancement accordingly.
